data/mods/Magiclysm/Spells/technomancer.json

```diff
@@ -480,5 +480,57 @@
     "max_range": 6,
     "energy_source": "MANA",
     "spell_class": "TECHNOMANCER"
+  },
+  {
+    "id": "technomancer_knifeshot",
+    "type": "SPELL",
+    "name": "Knifeshot",
+    "description": "This spell generates a magnetic field around a knife, before launching it at high speed.",
+    "valid_targets": [ "hostile", "ally" ],
+    "flags": [ "CONCENTRATE", "SILENT", "SOMATIC" ],
+    "components": "spell_components_knifeshot",
+    "effect": "projectile_attack",
+    "spell_class": "TECHNOMANCER",
+    "energy_source": "BIONIC",
+    "difficulty": 5,
+    "max_level": 20,
+    "base_casting_time": 600,
+    "casting_time_increment": -15,
+    "final_casting_time": 300,
+    "base_energy_cost": 50,
+    "min_damage": 30,
+    "max_damage": 70,
+    "damage_increment": 2.0,
+    "damage_type": "stab",
+    "min_range": 10,
+    "max_range": 20,
+    "range_increment": 0.5,
+    "min_pierce": 3,
+    "max_pierce": 10,
+    "pierce_increment": 0.35,
+    "learn_spells": { "technomancer_knifeshot_superior": 20 }
```

Comment:
i don't seem to be gaining the new spell when i get to level 20.

Reply:
This also applies to the Acid Resistance aura not giving Greater Acid Resistance anymore, I'm not sure how it's breaking.

Reply:
ok i will investigate separately and won't let it block this PR.